modules/client/examples/06-multisig-client/08-get-plugin-settings.ts

```diff
@@ -0,0 +1,34 @@
+/* MARKDOWN
+### Loading the list of members (multisig plugin)
+*/
+import {
+  Context,
+  ContextPlugin,
+  MultisigClient,
+  MultisigPluginSettings,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an multisig client
+const client = new MultisigClient(contextPlugin);
+
+const daoAddressorEns = "0x12345...";
+
+const settings: MultisigPluginSettings = await client.methods
+  .getPluginSettings(daoAddressorEns);
+console.log(settings);
```

Comment:
Isn't it `pluginSettings`? everywhere

Reply:
What do you mean? Is about the types names `MultisigPluginSettings` is defined but then I use som aliases to make it more readable, I can delete them if you want